socorro/processor/signature_utilities.py

```diff
@@ -243,17 +245,45 @@ def _do_generate(self,
         if sentinel_locations:
             source_list = source_list[min(sentinel_locations):]
 
+        # Get all the relevant frame signatures.
         new_signature_list = []
         for a_signature in source_list:
+            # If the signature matches the irrelevant signatures regex,
+            # skip to the next frame.
             if self.irrelevant_signature_re.match(a_signature):
                 continue
+
+            # If the signature matches the trim dll signatures regex,
+            # rewrite it to remove all but the module name.
+            if self.trim_dll_signature_re.match(a_signature):
+                try:
+                    a_signature = a_signature[:a_signature.index('@')]
```

Comment:
Normally you do `a_signature = a_signature.split('@')[0]` which does the same thing but without having to compute the index first. And then you don't need the `try:except ValueError` blocks.